client/reader/stream/index.jsx

```diff
@@ -516,7 +515,6 @@ class ReaderStream extends Component {
 					renderItem={ this.renderPost }
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 					className="stream__list"
-					tags={ tags }
```

Comment:
This isn't specifically related to the PR, but was inserting `tags="[object Object]"` into the dom and doesn't seem to have a specific purpose, so I removed it.